clusters/app.ci/ci-search/deploy.yaml

```diff
@@ -0,0 +1,108 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: ci-search
+
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: ci-search
+  namespace: ci-search
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - name: latest
+    importPolicy:
+      scheduled: true
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/ci/ci-search:latest
+
+# deploy the controller
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: search
+  namespace: ci-search
+spec:
+  selector:
+    app: search
+  ports:
+  - port: 80
+    targetPort: 8080
+
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: ci-search
+  namespace: ci-search
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      app: search
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: search
+  namespace: ci-search
+  annotations:
+    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"ci-search:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"web\")].image"}]'
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: search
+  serviceName: ""
+  template:
+    metadata:
+      labels:
+        app: search
+    spec:
+      automountServiceAccountToken: false
+      volumes:
+      - name: bugzilla-token
+        secret:
+          secretName: bugzilla-credentials-bugzilla.redhat.com
```

Comment:
Need to put it into BW.

/hold

Reply:
https://coreos.slack.com/archives/CHY2E1BL4/p1594228009192200

The secret is already in BW.
What is done here is put it into the secret tool.

/hold cancel